python/src/HSICStatImplementation_doc.i.in

```diff
@@ -32,9 +37,19 @@ covarianceMatrixY : :class:`~openturns.CovarianceMatrix`
 weightMatrix : :class:`~openturns.Matrix`
     A weight matrix used for the statistic.
 
+weights : :class:`~openturns.Point`
+    A weight used for the statistic.
+
 Returns
 -------
-hsicIndex : the HSIC index of the two :class:`~openturns.Sample`."
+hsicIndex : the HSIC index of the two :class:`~openturns.Sample`.
+
+Notes
+-----
+
+The first usage is deprecated.
```

Comment:
it would have been easier to not distinguish the name of the weight argument:
```
weights : :class:`~openturns.Point` or :class:`~openturns.Matrix` (deprecated)
```

Reply:
Good point, I'll fix it